query/src/main/java/org/divy/common/bo/query/operator/comparison/OperatorFactory.java

```diff
@@ -0,0 +1,52 @@
+package org.divy.common.bo.query.operator.comparison;
+
+import org.divy.common.bo.query.operator.And;
+import org.divy.common.bo.query.operator.Not;
+import org.divy.common.bo.query.operator.Operator;
+import org.divy.common.bo.query.operator.Or;
+import org.divy.common.bo.query.operator.comparison.impl.*;
+import org.divy.common.bo.query.operator.impl.AndImpl;
+import org.divy.common.bo.query.operator.impl.NotImpl;
+import org.divy.common.bo.query.operator.impl.OrImpl;
+
+public class OperatorFactory
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)